ferrocene/tools/pull-subtrees/subtrees.yml

```diff
@@ -26,10 +26,10 @@
   repo: rust-lang/libc
   ref: libc-0.2
   after:
-    - update-cargo-lock
+    - update-cargo-lock libc library
 
 - path: ferrocene/library/backtrace-rs
   repo: rust-lang/backtrace-rs
   ref: master
   after:
-    - update-cargo-lock
+    - update-cargo-lock backtrace library
```

Comment:
We aren't yet using the subtree in `library` I think, but we will likely do that soon

Reply:
actually nevermind, this is a patch dependence, it doesn't even show in the library lock file